### PR TITLE
Add concourse and FAKE build artifacts

### PR DESCRIPTION
**Pull request by jebbinSCL**

### CODE REVIEW

File: githooks/commit-msg [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# regex to validate in commit msg
+commit_regex='(BRITECH-[0-9]+)'
+error_msg="Aborting commit. Your commit message is missing a JIRA Issue ('BRITECH-1111')"
+
+if ! grep -iqE "$commit_regex" "$1"; then
+    echo "$error_msg" >&2
+    exit 1
+fi
```
Comment (lclarksl) [inline githooks/commit-msg]: Are we still using these git hooks? I thought we'd decided one pull request per task?

File: concourse/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,7 @@
+FROM mono
+
+COPY binaries/ service/
+
+EXPOSE 9000
```
Comment (lclarksl) [inline concourse/Dockerfile]: This should be 9003

**Comment by lclarksl** [comment]
:shipit: after git hooks decided

File: .gitignore [hunk 3]
```diff
@@ -234,3 +234,24 @@ _Pvt_Extensions
...
+build/output/
+
+# Nuget stuff (can be install with build.sh)
+.nuget/
+
+# Ubuntu trash folder
+.Trash-*
+
+# This file needs to have the parameters required by the pipeline
+concourse/credentials.yml
```
Comment (lclarksl) [inline .gitignore]: Do we have a credentials file?

**Comment by ianlovell** [comment]
:shipit:

**Pull request merged**